README.md

```diff
@@ -1,3 +1,45 @@
 # multicaller
 
 Helper for using multicall to aggregate onchain data calls into a single RPC call.
```

Comment:
```suggestion
Helper for using multicall to aggregate onchain data calls into a single RPC call.
## Features
* Uses multicall3, allowing for failing calls on a per-call basis
* Iteratively breaks up a call bundle if it's too large for the endpoint
* Uses native web3py contract functions rather than requiring users to [write the function ABI for each call](https://github.com/banteg/multicall.py/blob/9117c552c6ae85acec8452961db0aac51119070b/examples/daistats.py#L77)
```